protocol/xmrmaker/net.go

```diff
@@ -75,6 +76,7 @@ func (inst *Instance) initiate(
 
 	symbol, err := pcommon.AssetSymbol(inst.backend, offer.EthAsset)
 	if err != nil {
+		_ = s.Exit()
```

Comment:
Would that error get logged somewhere else, or are we just swallowing it? We could log the one that is not propagated up, or we could do this:
```
	if exitErr := s.Exit(); err != nil {
		return nil, multierror.Append(err, exitErr)
	}
```

Reply:
we're swallowing it but in this case I think it's fine, the swap is being aborted anyways since initiation failed. can add multierror if you think that's useful

Reply:
will add a log in `s.Exit()` for now